net/network/netem-control/files/sbin/netem-control.lua

```diff
@@ -21,29 +23,38 @@ function setRules (operation)
     local qdisc_parent = 0
     local errors = 0
 
+    if ( operation == "del") then
+        --remove all rules associated to previously added interfaces
+        for k,v in ipairs(intfcfg) do
+            tc_str = "tc qdisc del dev "..v.." root"
+            exec(tc_str)
+        end
+        table.remove(intfcfg)
+        return
+    end
```

Comment:
I added this code to take care of the `(operation == "del")` condition. This should make all further `tc qdisc del dev iface root` references redundant and the reason I commented out those sections.